tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateAdditionalLikely.java

```diff
@@ -0,0 +1,297 @@
+import com.google.common.collect.TreeMultimap;
+import com.ibm.icu.lang.UScript;
+import com.ibm.icu.text.UnicodeSet;
+
+public class GenerateAdditionalLikely {
+
+    private static final CLDRConfig CLDR_CONFIG = CLDRConfig.getInstance();
+    private static final Splitter UNDERBAR = Splitter.on('_');
+    private static final Splitter TAB_SPLITTER = Splitter.on('\t');
+
+    static class ScriptRegion {
+        Multimap<Pair<String, String>, String> data = TreeMultimap.create();
+
+        public void put(String script, String region, String source) {
+            data.put(Pair.of(script, region), source);
+        }
+
+        @Override
+        public String toString() {
+            return data.toString();
+        }
+    }
+
+    static Factory factory = CLDR_CONFIG.getExemplarsFactory();
+    static CLDRFile english = CLDR_CONFIG.getEnglish();
+    static LanguageTagParser ltp = new LanguageTagParser();
+    static Validity validity = Validity.getInstance();
+
+    static final Set<String> LANGUAGE_REGULAR = validity.getStatusToCodes(LstrType.language).get(Status.regular);
+    static final Set<String> SCRIPT_REGULAR = validity.getStatusToCodes(LstrType.script).get(Status.regular);
+    static final Set<String> REGION_REGULAR = validity.getStatusToCodes(LstrType.region).get(Status.regular);
+
+    static final Set<String> LIKELY_SPECIALS = ImmutableSet.of("in", "iw", "ji", "jw", "mo");
+    static final Set<String> FIX_VALIDITY = ImmutableSet.of("Zanb");
+    static final Set<String> FIX_COUNTRY = ImmutableSet.of("yi");
+
+    public static boolean isOk(String lang, String script, String region, Map<LstrType, Status> errors) {
+        errors.clear();
+        if (!LIKELY_SPECIALS.contains(lang)) {
+            check(LstrType.language, lang, errors);
+        }
+        if (!FIX_VALIDITY.contains(script)) {
+            check(LstrType.script, script, errors);
+        }
+        if (region.equals("001") && Iso639Data.getType(lang) == Type.Constructed){
+            // ok
+        } else {
+            check(LstrType.region, region, errors);
+        }
+        return errors.isEmpty();
+    }
+
+    public static void check(LstrType lstrType, String lang, Map<LstrType, Status> errors) {
+        final Status status = validity.getCodeToStatus(lstrType).get(lang);
+        if (status != Status.regular) {
+            errors.put(lstrType, status);
+        }
+    }
+
+    public static void main(String[] args) {
+        list("de en es es-419 fr hr it nl pl pt-BR pt-PT vi tr ru ar th ko zh-CN zh-TW ja ach af ak az ban su xx-bork bs br ca ceb cs sn co ht cy da yo et xx-elmer eo eu ee tl fil fo fy gaa ga gd gl gn xx-hacker ha haw bem ig rn id ia xh zu is jw rw sw tlh kg mfe kri la lv to lt ln loz lua lg hu mg mt mi ms pcm no nn nso ny uz oc om xx-pirate ro rm qu nyn crs sq sk sl so st sr-ME sr-Latn fi sv tn tum tk tw wo el be bg ky kk mk mn sr tt tg uk ka hy yi iw ug ur ps sd fa ckb ti am ne mr hi bn pa gu or ta te kn ml si lo my km chr");
```

Comment:
move constant to top and document
(ideally in a data file if this is going to be rerun)